### DISCUSSION TITLE
Broken code application

### DISCUSSION THREAD

**Discussion by reneqvist** [shibayan/appservice-acmebot#310] [General]
Cant recommend using this app, it fails on basic apps, installation guide is out-of-date and documentation is poor

**Comment by shibayan** [comment]
Thanks for the unproductive discussion. There is no denying that using this tool requires knowledge and experience with Azure.

You should be able to contribute as it is open source and pull requests are always accepted.

**Reply by shibayan** [reply]
It is not motivating for me to write only uninformative and unhelpful comments when you are unable to have constructive discussions such as questions and improvements. To put it bluntly, it's a distraction.

**Reply by reneqvist** [reply]
I'm sorry you feel hurt by my comment. But my honest opinion about the project stands. Being on the ACME Client Implementation list on the official LetsEncrypt obliges to a better quality project, in my eyes anyway.

**Reply by reneqvist** [reply]
I have looked through different posts here where people have challenges, and you just turn them down with no helpfulness.

**Reply by shibayan** [reply]
I don't disagree with your opinion, but you don't seem to have any intention of making any specific point at all.

Most of the users, except you, are using it without any problems and, unlike you, occasionally give us meaningful feedback. There have also been multiple contributions to the documentation, which has been improved.

Don't forget that this is Open source. This means that anyone can participate in the improvement.

**Reply by shibayan** [reply]
OSS is based on community-based, best-effort support. If you need more extensive support, I can provide you with a paid plan. (Of course, it will be expensive.)

**Reply by reneqvist** [reply]
My point is that you are listed on the official LetsEncrypt site, which in my opinion sends a signal as being a quality project. What I'm reading on the project discussions, is that people are experiencing unexplainable errors, for which you reply "no support".

**Reply by shibayan** [reply]
You seem not to have read this sentence. This page is just a list of clients that support ACME protocol. The signal is just a misunderstanding on your part.

> The ACME clients below are offered by third parties. Let’s Encrypt does not control or review third party clients and cannot make any guarantees about their safety or reliability.

https://letsencrypt.org/docs/client-options/

**Reply by reneqvist** [reply]
Maybe so. all the best with your project.

**Comment by luis-cortina** [comment]
@shibayan I'll jump in here and say that this project is a life saver for me personally, I don't get @reneqvist comment or approach to discuss this tool. 

I would move on from this and spend time on more fruitful things.

**Reply by shibayan** [reply]
Thank you. I'm really glad you said that.

Rest assured that development will continue to be active!
